test/smoke/src/areas/positron/top-action-bar/interpreter-dropdown.test.ts

```diff
@@ -0,0 +1,187 @@
+					return (
+						contents.some((line) => line.includes('preparing for restart')) &&
+						contents.some((line) => line.includes('restarted'))
+					);
+				});
+
+				// Wait for the console to be ready
+				await positronConsole.waitForReady('>>>', 10000);
+
+				// The interpreter dropdown should show the expected running indicators
+				expect(
+					await interpreterDropdown.primaryInterpreterShowsRunning('Python')
+				).toBe(true);
+			});
+
+			it('R interpreter starts and shows running [C707214]', async function () {
+				// Start an R interpreter using the interpreter dropdown
+				await expect(
+					async () => await interpreterDropdown.selectInterpreter('R', desiredR)
+				).toPass({ timeout: 15_000 });
+
+				// Wait for the console to be ready
+				await positronConsole.waitForReady('>', 10000);
+
+				// The interpreter selected in the dropdown matches the desired interpreter
+				const interpreterInfo =
+					await interpreterDropdown.getSelectedInterpreterInfo();
+				expect(interpreterInfo?.version).toBeDefined();
+				expect(interpreterInfo!.version).toContain(desiredR);
+				expect(interpreterInfo!.path).toBeDefined();
+
+				// The interpreter dropdown should show the expected running indicators
+				expect(
+					await interpreterDropdown.primaryInterpreterShowsRunning(
+						interpreterInfo!.path
+					)
+				).toBe(true);
+			});
+
+			it('R interpreter stops and shows inactive [C707215]', async function () {
+				// NOTE: This test is dependent on 'R interpreter starts and shows running' having run successfully
+
+				// Stop the active R interpreter
+				expect(async () => {
+					await interpreterDropdown.stopPrimaryInterpreter('R');
+				}).toPass({ timeout: 15_000 });
+
+				// The console should indicate that the interpreter is shutting down
+				await positronConsole.waitForConsoleContents((contents) =>
+					contents.some((line) => line.includes('shut down successfully'))
+				);
+
+				// The interpreter dropdown should no longer show the running indicators
+				expect(
+					await interpreterDropdown.primaryInterpreterShowsInactive('R')
+				).toBe(true);
+			});
+		});
+
+		/**
+		 * NOTE: This describe block MUST RUN LAST, after any of the other tests in this suite.
+		 * This unfortunately cannot be run in the 'after' block of the previous describe block,
+		 * as Positron closes before or while the after block is running.
+		 */
+		describe('Interpreter Dropdown in Top Action Bar - cleanup', async () => {
+			// Shared before/after handling
+			installAllHandlers(logger);
+
+			let app: Application;
+			let userSettings: PositronUserSettingsFixtures;
+
+			before(async function () {
+				app = this.app as Application;
+				userSettings = new PositronUserSettingsFixtures(app);
+			});
+
+			it('Unset user settings', async function () {
+				// Unset the user setting to re-enable automatic startup of interpreters
+				await userSettings.unsetUserSettings();
+			});
+		});
+	});
```

Comment:
Clears the user settings after the tests have run, to restore the original behaviour

Reply:
this has been removed